### ISSUE TITLE
Add tutorials section

### ISSUE THREAD

**Issue by hallvaaw** [hallvaaw/awesome-biological-image-analysis#11]
It would be nice to have a section with links to tutorials, introductions, video lessons etc. for biological image analysis.

**Comment by LSeu-Open** [comment]
I can try to add some, i have link in mind for tutorials from few projects that are linked in their readme.

**Comment by LSeu-Open** [comment]
here is an exemple from Cellpose explaining how to train your own prediciton model : https://www.youtube.com/watch?v=3Y1VKcxjNy4 . Found on the readme.md

Also write some for my team i can share it here.

**Comment by hallvaaw** [comment]
Sounds good. I am mainly thinking of generalized tutorials for image analysis eg. how to preprocess images, image segmentation, particle analysis and so on.
Perhaps some of the entries in this list are worth a mention: https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html#written-resources

**Comment by LSeu-Open** [comment]
Yes this is a very valuable source. It may need differents md files from the readme to keep clarity.


On a another hand i can think of adding link to project's documentation pages/Usage tutorials (Video-notebooks) for each project (Maybe not all people are into looking deep in documentations...)
